README.md

```diff
@@ -157,6 +157,21 @@ All columns are sortable by default, but sorting can be disabled by setting `sor
     { key: 'year', label: 'Year', sortable: false }
 
 
+##### Table Header Column CSS Class
+
+To set the css class for table header **&lt;th&gt;**, use the optional *thclass* key. This attribute can be a String or a Function.
```

Comment:
Oh, I missed the list with all key for settings in README for that. It also should be added there.